lib/popover.js

```diff
@@ -43,7 +43,7 @@ const usePositionAdjustment = () => {
 
 const PopoverWrap = styled.div`
   position: absolute;
-  width: auto;
+  width: max-content;
```

Comment:
This may break popovers in IE/Edge since this value isn't supported in those browsers.

Reply:
ugh I can't believe I didn't check that 🤦‍♀ let me double check that it does in fact work better than auto, and if so we can poke at it further!

Reply:
ok, the one place I'm currently finding an issue is team-user-pop. possibilities:

as current:
<img width="166" alt="Screen Shot 2019-12-18 at 11 56 00 AM" src="https://user-images.githubusercontent.com/3011231/71106494-78434000-218d-11ea-8c71-0e70f8579a1d.png">

`max-content`:
<img width="360" alt="Screen Shot 2019-12-18 at 11 56 21 AM" src="https://user-images.githubusercontent.com/3011231/71106519-81cca800-218d-11ea-93a6-90956b9b3968.png">

I think what would work is to make it so that if a Button has an Icon, make it so that there isn't a break between the text and the image. We can't do `white-space: nowrap` on the Button though, because if there's lots of text (like in the example above), we do need it to wrap.
Thoughts?

Reply:
I see. I thought that adding a min-width to popovers was a way to fix this (the editor has static width popovers), but that would break the tiny pops that we use for collection options. In the editor, we override the pop width in cases like the file options pop.

Maybe text-wrapping could be changed from the default, to an option that's false by default. I think most of our buttons are only a few words anyways. We could toggled the text-wrap on in specific cases where we know we're using really long copy — plus, using really long copy should probably be avoided.